corehq/apps/case_search/views.py

```diff
@@ -60,5 +60,5 @@ def post(self, request, *args, **kwargs):
             addition = CaseSearchQueryAddition.objects.get(id=query_addition, domain=self.domain)
             new_query = merge_queries(search.get_query(), addition.query_addition)
             search = search.set_query(new_query)
-        search_results = search.run().raw_hits
-        return json_response({'values': search_results})
+        search_results = search.size(CASE_SEARCH_MAX_RESULTS).run().raw_hits
+        return json_response({'values': search_results, 'count': search.count()})
```

Comment:
This will execute the query twice. You can get the count from the some results as the search results:
```
results = search.size(...).run()
search_results = results.raw_hits
count = results.total
```